packages/client-core/src/components/ViewerInteractions/index.tsx

```diff
@@ -74,10 +75,15 @@ export const ViewerInteractions = () => {
     )
   }
 
+  const viewerMenuState = getMutableState(ViewerMenuState)
+
   return (
     <div style={{ opacity: 1 - loadingScreenOpacity.value }} className="relative h-dvh w-full p-6">
       <div className="pointer-events-auto absolute left-0 top-0 h-fit w-full pt-[inherit]">
-        <MediaIconsBox />
+        <div className="flex items-center justify-between px-6">
+          <MediaIconsBox />
+          {viewerMenuState.userMenus.ecommerce && viewerMenuState.menuComponents.get(NO_PROXY).ecommerce}
```

Comment:
we should add all the menucomponents in a generic way
all of them should go to the top right portion of the screen/viewer